app/models/pacbio/run.rb

```diff
@@ -34,6 +34,12 @@ class Run < ApplicationRecord
 
     scope :active, -> { where(deactivated_at: nil) }
 
+    accepts_nested_attributes_for :plate
+
+    # This will return an empty list
+    # If well data is required via the run, use ?include=plate.wells
+    attr_reader :well_attributes
```

Comment:
Hm I'm not sure. It fails with `"exception":"undefined method 'well_attributes'` if this attr_reader is not provided.

Not sure what to do here, because we don't want to return all the well info as default, I don't think. Instead the well info can be explicitly included.